tox.ini

```diff
@@ -6,14 +6,14 @@ envlist = py{37,38}-{linux,windows}
 python =
     3.7: py37
     3.8: py38
-    
+
```

Comment:
Also see `cellfinder-core`. I don't know why this file exists.

Reply:
will raise another issue

Reply:
https://github.com/brainglobe/cellfinder/issues/194